lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -122,7 +122,8 @@ def test_404(self):
         )
 
     def test_get_success(self):
-        response = self.client.get(self.url)
+        with check_mongo_calls(2):
+            response = self.client.get(self.url)
```

Comment:
How many topics does this test create?
What does this call to `check_mongo_calls` on master return?  That is, how many queries prior to this change?